pkg/kubelet/server/remotecommand/httpstream.go

```diff
@@ -116,7 +116,10 @@ func createStreams(req *http.Request, w http.ResponseWriter, opts *Options, supp
 
 	if ctx.resizeStream != nil {
 		ctx.resizeChan = make(chan remotecommand.TerminalSize)
-		go handleResizeEvents(ctx.resizeStream, ctx.resizeChan)
+		go func() {
+			handleResizeEvents(ctx.resizeStream, ctx.resizeChan)
+			close(ctx.resizeChan)
```

Comment:
would it make sense to move this into `handleResizeEvents` as a `defer close(channel)`?

Reply:
I was thinking that it is easy to check if we make `close(ctx.resizeChan)` closed to `make(chan remotecommand.TerminalSize)` statement. But your suggestion sgtm. updated it.

Reply:
I am fine with it in the present location.  In reviewing, I had initially looked for where the channel was created initally as well, but its fine here.